service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java

```diff
@@ -29,6 +33,32 @@ public static ApiCreateWorkspaceRequestBody createWorkspaceRequestBody() {
     return createWorkspaceRequestBody(ApiWorkspaceStageModel.MC_WORKSPACE);
   }
 
+  public static Workspace createWorkspace(
```

Comment:
could this be something like `buildWorkspace` to make it clearer this is building an object, not creating/writing a workspace DB record?

Reply:
sure